utils/kata-manager.sh

```diff
@@ -583,29 +584,36 @@ configure_containerd()
 install_kata()
 {
 	local requested_version="${1:-}"
+	local kata_tarball="${2:-}"
 
 	local project="$kata_project"
 
-	local version_desc="latest version"
-	[ -n "$requested_version" ] && version_desc="version $requested_version"
-
-	info "Downloading $project release ($version_desc)"
-
-	local results
-	results=$(github_download_package \
-		"$kata_releases_url" \
-		"$requested_version" \
-		"$project")
-
-	[ -z "$results" ] && die "Cannot download $project release file"
-
-	local version
-	version=$(echo "$results"|cut -d: -f1)
-
-	local file
-	file=$(echo "$results"|cut -d: -f2-)
+	local version=""
+	if [ -z "$kata_tarball" ]
+	then
+		local version_desc="latest version"
+		[ -n "$requested_version" ] && version_desc="version $requested_version"
+	
+		info "Downloading $project release ($version_desc)"
+	
+		local results
+		results=$(github_download_package \
+			"$kata_releases_url" \
+			"$requested_version" \
+			"$project")
+	
+		[ -z "$results" ] && die "Cannot download $project release file"
+	
+		version=$(echo "$results"|cut -d: -f1)
+
+		[ -z "$version" ] && die "Cannot determine $project resolved version"
+
+		local file
+		file=$(echo "$results"|cut -d: -f2-)
+	else
+		file="$kata_tarball"
```

Comment:
Nit: This works fine but you end up with a message like this (with my hash comments below the output):

```
Installing Kata Containers release  from /home/james/data/kata/kata-static-3.2.0-amd64.tar.xz
#                                 ~~
#                                 |
#                            no version
```

How about we change the `info` call below to something like the following to resolve that:

```bash
if [ -n "$kata_tarball" ]
then
    info "Installing $project release from '$file'"
else
    info "Installing $project release $version from '$file'"
fi
```